src/Generators/Microsoft.Gen.Logging/Emission/Emitter.Method.cs

```diff
@@ -99,7 +99,7 @@ private void GenLogMethod(LoggingMethod lm)
 
         var lambdaStateName = PickUniqueName("s", lm.TemplateToParameterName.Select(kvp => kvp.Key));
 
-        OutLn($"static ({lambdaStateName}, {exceptionLambdaName}) =>");
+        OutLn($"[{GeneratorUtilities.GeneratedCodeAttribute}] static ({lambdaStateName}, {exceptionLambdaName}) =>");
```

Comment:
Consider also adding the lambda return type to the generated lambda as well. Has a positive impact on compiler performance.

Reply:
It's fairly new syntax.